src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -567,4 +567,92 @@
   <data name="SwitchSelectionEndpointCommandKey" xml:space="preserve">
     <value>Switch selection endpoint</value>
   </data>
+  <data name="ColorSelection_allMatches" xml:space="preserve">
+    <value>, all matches</value>

```

Comment:
```suggestion
    <value>all matches</value>
```
nit: let's keep the grammar responsibility in the code in ActionArgs.cpp

so `fmt` will be responsible for adding the `,` but "all matches" can be localized on its own here

Reply:
Wouldn't we have to translate the comma? Japanese for instance uses wide commands ("、").

Reply:
I'd agree with you but we're already littered with the commas hahaha

https://github.com/microsoft/terminal/blob/64bcc0bd25fd2543f4a71dfc5beb1063b807851b/src/cascadia/TerminalSettingsModel/ActionArgs.cpp#L58-L80

Might as well be consistent